hack/release.sh

```diff
@@ -7,12 +7,12 @@
 
 set -e
 echo "Logging into container registry $IMAGE_HOST"
-echo "$REGISTRY_PASSWORD" | $CONTAINER_RUNTIME login -u "$REGISTRY_USERNAME" --password-stdin "$IMAGE_HOST"
+echo "$REGISTRY_PASSWORD" | ko login -u "$REGISTRY_USERNAME" --password-stdin "$IMAGE_HOST"
 
-echo "Building container image $IMAGE_HOST/$IMAGE:$TAG"
-make build-image
+echo "Building container image"
 
-echo "Pushing container image to $IMAGE_HOST"
-make push-image
+# Using defaults, this pushes to:
+# quay.io/shipwright/shipwright-operator/github.com/shipwright-io/build/cmd/manager:latest
+KO_DOCKER_REPO="$IMAGE_HOST/$IMAGE" ko resolve -t "$TAG" -P -R -f deploy/ > release.yaml
```

Comment:
From the documentation of [`ko resolve`](https://github.com/google/ko#ko-resolve), I am not sure if this step is triggering a build under the cover or only creates the yaml ? In any case, I guess this will need to go into the Makefile to use the GO_FLAGS defined there.

Reply:
`ko resolve` scans for `ko://` references, builds those paths, and replaces the placeholders with built image references by digest in the concatenated output YAML, which it sends to stdout (here, to `release.yaml`). So yes, if we want to pass through GOFLAGS we'd need to do that here too.

Reply:
Yeah, I think we want the `GO_FLAGS` from the makefile to be applied here as well.